src/main/scala/com/cloudant/clouseau/IndexService.scala

```diff
@@ -633,7 +605,8 @@ class IndexService(ctx: ServiceContext[IndexServiceArgs]) extends Service(ctx) w
     case e: ParseException =>
       ('error, ('bad_request, e.getMessage))
     case e =>
-      ('error, e.getMessage)
+      val stackTrace = (for (t <- e.getStackTrace()) yield t.toString()).mkString("\n")
+      ('error, (e.getClass, e.getMessage, stackTrace))
```

Comment:
Sometimes a `NullPointerException` was thrown and it was very hard to find the source of that because it did not even have an associated message.  It helped me the work on this change, and I felt it could only do good if it is left there as is.

Reply:
This one breaks the message format, so I decided to remove it.